<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BasharPC" failures="1" tests="3" name="Surefire test" time="32.142" errors="0" timestamp="2024-06-18T08:58:28 PDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.274"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.257">
    <failure type="java.lang.AssertionError" message="expected [ready] but found [finished]">
      <![CDATA[java.lang.AssertionError: expected [ready] but found [finished]
at stepDefinitions.FileEncryptionAndDecryptionTest.The_user_clicks_the_Decrypt_button_to_encrypt_the_file_1(FileEncryptionAndDecryptionTest.java:195)
at âœ½.The user clicks the Decrypt button to decrypt the file correctly(file:///C:/Users/basha/Downloads/crypto-testing/src/test/java/features/FileEncryptionAndDecryptionTest.feature:19)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.264"/>
</testsuite> <!-- Surefire test -->
